.drone.yml

```diff
@@ -56,6 +56,7 @@ steps:
     commands:
       - /setup_sandbox.sh
       - enonic project build
+      - sleep 60
```

Comment:
Jeg ville plassert denne i "deploy" step, ikke i "build & test". For jeg tror ikke den har noen særlig virkning her.